### ISSUE TITLE
Type Error

### ISSUE THREAD

**Issue by thefrosty** [cedaro/satispress#126]
Been receiving these notices via WordPress' new fatal error detection email notifications. It's a site that I know gets zero traffic, so not sure if these are triggered around possible cron events.
Site info and error listed below.

WordPress version `5.3.2`
Current theme: `Twenty Nineteen (version 1.4)`
Current plugin: `SatisPress (version 0.5.0)`
PHP version `7.3.14`

Error Details
=============
```
An error of type E_ERROR was caused in line 193 of the file /../wp-content/plugins/satispress/src/Provider/Authentication.php. 
Error message: Uncaught TypeError: ltrim() expects parameter 1 to be string, 
null given in /../wp-content/plugins/satispress/src/Provider/Authentication.php:193
Stack trace:
#0 /../wp-content/plugins/satispress/src/Provider/Authentication.php(193): ltrim(NULL, '/')
#1 /../wp-content/plugins/satispress/src/Provider/Authentication.php(164): SatisPress\Provider\Authentication->get_request_path()
#2 /../wp-content/plugins/satispress/src/Provider/Authentication.php(75): SatisPress\Provider\Authentication->is_satispress_request()
#3 /../wp-content/plugins/satispress/vendor/cedaro/wp-plugin/src/AbstractPlugin.php(180): SatisPress\Provider\Authentication->register_hooks()
#4 /../wp-content/plugins/satispress/satispress.php(74): Cedaro\WP
```

**Comment by bradyvercher** [comment]
Thanks for the report, @thefrosty. I think this might be fixed by 0905e7fb, so I'll need to get a new release pushed out to include that.

**Comment by thefrosty** [comment]
Looks like this can be closed now that v0.5.1 https://github.com/cedaro/satispress/releases/tag/v0.5.1 is released.

**Comment by bradyvercher** [comment]
Thanks! I meant to take care of this yesterday, but it got lost amongst the tabs.
